java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/utils/UrlSample.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.arrow.driver.jdbc.utils;
+
+import org.apache.arrow.util.Preconditions;
+
+/**
+ * Class for storing sample JDBC URLs. Used for testing.
+ *
+ * @see org.apache.arrow.driver.jdbc.utils.BaseProperty
+ * @deprecated not updatable to match dinamic server allocation.
+ */
+@Deprecated
+public enum UrlSample {
```

Comment:
Why are we adding a new deprecated class?

Reply:
We're cleaning up deprecated classes/methods.